app/api/utils.py

```diff
@@ -451,6 +471,67 @@ def paint(self, documents):
         return res
 
 
+class SpacyPainter(JSONPainter):
+
+    def paint(self, documents):
+        data = super().paint(documents)
+        res = []
+
+        label_ids = {a['label'] for d in data for a in d['annotations']}
+        label_texts = {l['id']: l['text'] for l in Label.objects.filter(id__in=label_ids).values('text', 'id')}
+
+        for d in data:
+            text = d['text']
+            annotations = d.pop('annotations')
+            entities = []
+            for a in annotations:
+                label_text = label_texts[a['label']]
+                entities.append([a["start_offset"], a["end_offset"], label_text])
+            res.append((text, {"entities": entities}))
+        return [res]
+
+
+class Color:
```

Comment:
This class was removed in https://github.com/doccano/doccano/pull/393. Looks like this is a merge artifact?